aten/src/ATen/native/quantized/cpu/Normalization.cpp

```diff
@@ -407,4 +489,8 @@ TORCH_LIBRARY_IMPL(quantized, QuantizedCPU, m) {
   m.impl(TORCH_SELECTIVE_NAME("quantized::batch_norm3d_relu"), TORCH_FN(q_batch_norm3d_impl<true>));
 }
 
+TORCH_LIBRARY_IMPL(onednn, CPU, m) {
+  m.impl(TORCH_SELECTIVE_NAME("onednn::qbatch_norm2d"), TORCH_FN(int8_batch_norm2d_cpu_impl));
```

Comment:
hmm, even though the op is put under "onednn" namespace but seems the implementation does not rely on onednn at all?

Reply:
Yes, that's true. It's because all these ops used for PT2E quantization are put under the onednn namespace, and they should only be used when using the X86Inductor backend. We decide to keep that practice. Do you have any concerns? Thanks.

Reply:
Perhaps we can separate the "op" namespace and the quantization backend such as "onednn". For op, "onednn" means that the implementation relies on the oneDNN library and don't have to put other quantized ops under "onednn" namespace. On the other hand, the quantization support with "onednn" quant backend can leverage ops implemented with oneDNN library and native ATen. What do you think?

Reply:
Thanks for the suggestion. It makes sense to me. How would you suggest we name the namespace? `x86` or still `quantized`? cc @leslie-fang-intel  @jerryzh168

Reply:
Hi @jerryzh168 @leslie-fang-intel Do you think it's ok to use `onednn` namespace even if the we don't use oneDNN library for implementation? If not, do you have any suggestions on name of the namespace? Thanks.

Reply:
btw, it could be a long-term refactoring work, not necessarily blocking this PR landing.

Reply:
it can be `x86` I think, since it's used in x86 inductor backend

Reply:
@jgong5 @jerryzh168 Thanks for your suggestions. I will keep this PR as is and refactor this part in a future PR.